lib/numbers.md

```diff
@@ -5,20 +5,20 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+Integers (whole numbers) and floating-point numbers (decimal points)
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+Addition, subtraction, multiplication, division, greater than, less than ...
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+The '+' operation works as a sum for numbers.  For Strings it simply adds the text to the previous text.
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+No it wont work. Ruby doesn't know how to add a number to a string.  You have to tell Ruby that the string is an actual number with the .to_i operation first. So it would be '20' .to_i / 4
```

Comment:
Just making sure you saw the above comment. I can approve this assignment once you address that question.